src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -478,6 +478,11 @@ configurationRegistry.registerConfiguration({
 			description: nls.localize('explorer.incrementalNaming', "Controls what naming strategy to use when a giving a new name to a duplicated Explorer item on paste."),
 			default: 'simple'
 		},
+		'explorer.autoCreateNewFolder': {
+			'type': 'boolean',
+			description: nls.localize('explorer.autoCreateNewFolder', "Wether to create folder when new file name input value ends with slash (`/`)."),
```

Comment:
Whether typo

Reply:
Emm, okay I guess. The code was removed, how did you leave the review?